src/main/resources/static/_shared.yml

```diff
@@ -3842,6 +3842,11 @@ components:
           format: date-time
         document:
           $ref: '#/components/schemas/AnyValue'
+        canBeDirectlyWithdrawn:
+          description: |
+            If true, the user making this request can withdraw this request for placement. 
+            If false, it may still be possible to indirectly withdraw this request for placement by withdrawing the application.
```

Comment:
Can you remind me (and maybe include in the commit msg) in which scenario a RfP is not individually withdraw-able but can be withdrawn as part of a withdrawal cascade?

-> I think I found the answer in the integration test:

> user created the placement application, but it's not submitted, so they can't withdraw

I've left another comment about whether this logic should be more encapsulated and thereby perhaps easier to find?

Reply:
Yes, that's a good example. See other comment for notes on encapsulation